NoSoft-SA/nspack
# Hotfix/batch preprint rebins fix

Summary of this change:
Bugfix to accommodate clients who batch preprint rebin labels and have the client rule `use_db_sequence_for_bin_asset_numbers` set to true.

CHANGELOG entry:

### Fixed
- Batch pre-printing a rebin label no longer causes `adding a duplicate` validation error.

How to test this code via the UI:
As a client with the client rule set to true go to Production | In Progress | active setups and try to print rebin labels. Be sure to try both singular as well as multiple and confirm that it still works as intended for clients who have the client rule set to false.
